### DISCUSSION TITLE
How to Create a COMMON.ld File for STM32F427, STM32F429, and Other Controllers

### DISCUSSION THREAD

**Discussion by amitx64** [bitcraze/discussions#1537] [Q&A]
Hi,

I'm curious about how to create or modify the **COMMON.ld** file for STM32 controllers other than the STM32F405, such as the STM32F427 and STM32F429.

**Comment by ataffanel** [comment]
I am not sure I understand the question. You can modify the COMMON.ld in the current firmware to fit another MCU, the modified version will then be used. Examples of these linker file can often be found in the STM32's SDKs.
